pkg/registry/rbac/rest/storage_rbac.go

```diff
@@ -34,11 +34,11 @@ import (
 	"k8s.io/apiserver/pkg/registry/rest"
 	genericapiserver "k8s.io/apiserver/pkg/server"
 	serverstorage "k8s.io/apiserver/pkg/server/storage"
+	"k8s.io/client-go/util/retry"
```

Comment:
If we have multiple consumers of this, I'd rather it go to the eventual util repo than the client repo.  Am I wrong?